pkg/exchange/okex/exchange.go

```diff
@@ -269,16 +269,22 @@ func (e *Exchange) SubmitOrder(ctx context.Context, order types.SubmitOrder) (*t
 		return nil, fmt.Errorf("unexpected length of order response: %v", orders)
 	}
 
-	orderRes, err := e.QueryOrder(ctx, types.OrderQuery{
-		Symbol:        order.Symbol,
-		OrderID:       orders[0].OrderID,
-		ClientOrderID: orders[0].ClientOrderID,
-	})
+	orderID, err := strconv.ParseUint(orders[0].OrderID, 10, 64)
 	if err != nil {
-		return nil, fmt.Errorf("failed to query order by id: %s, clientOrderId: %s, err: %w", orders[0].OrderID, orders[0].ClientOrderID, err)
-	}
-
-	return orderRes, nil
+		return nil, fmt.Errorf("failed to parse response order id: %w", err)
+	}
+
+	timeNow := time.Now()
```

Comment:
We specify the time after the network operation, which can take seconds or longer. Running time.Now() upon entering this method would yield a more accurate timestamp